packages/components/src/components/popover/popover.scss

```diff
@@ -71,3 +71,7 @@
         pointer-events: none;
     }
 }
+
+.react-tiny-popover-cursor-option {
```

Comment:
can we add this as a nested class instead?

Reply:
@balakrishna-binary, no, because this class added to <div>, which parent is <body>.
![Screenshot 2021-10-05 at 09 53 44](https://user-images.githubusercontent.com/88426229/135974631-08a8b445-f543-4456-88bc-3cf7ef1e12a0.png)

Reply:
@maryiafrantsava-deriv you can still use nested class and use `@at-root` rule for it to insert the CSS at the root level. 
- https://sass-lang.com/documentation/at-rules/at-root